### DISCUSSION TITLE
Setting input `autocomplete` option globally with DI

### DISCUSSION THREAD

**Discussion by hakimio** [taiga-family/taiga-ui#7473] [Q&A] [answered]
Is it possible to set input `autocomplete` option globally using some DI provider?
Having to put `<input autocomplete="new-password" tuiTextfield />` boilerplate in every Taiga input is quite annoying.

**Comment by waterplea** [comment, marked as answer]
No, there's no such option. You can create your own directive for that and use the same selector tuiTextfield.
